http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/NonLeoMessageSubscriber.scala

```diff
@@ -86,6 +86,7 @@ class NonLeoMessageSubscriber[F[_]: Timer](gkeAlg: GKEAlgebra[F],
     for {
       ctx <- ev.ask
       clusterId = msg.clusterId
+      _ <- logger.info(ctx.loggingCtx)(s"Beginning to delete kubernetes cluster ${clusterId} in project ${msg.project} from NonLeoMessageSubscriber")
```

Comment:
not sure if this is necessary. If a cluster is deleted from `zombieMonitor`, we should have `deletedFrom` field indicating that (if cron job is not doing that currently, we should update there)

Reply:
oh nvm, cron job just sends message to leo, so I guess we can really differentiate